### PR TITLE
Custom theme

### PR DESCRIPTION
**Pull request by asleepingpanda**
## DESCRIPTION

Exposes the core pieces of the Apollos UI Theme in the API for us to be able to create custom content specific themes from Rock.

### CODE REVIEW

**Review by jordanwade — COMMENTED** [review]
@asleepingpanda could any of these changes break anything? Looks like you are just exposing more color options in the theme. I just don't have enough context to know if there is something I should test before giving approval on this PR.

**Comment by asleepingpanda** [comment]
@jordanwade _could_ it? Possibly. I don't think so, though, as the change is intended to be strictly additive so that it can be used in the next client side update.

File: src/data/theme/index.js [hunk 1]
```diff
@@ -19,14 +19,31 @@ export const schema = gql`
...
+    wordOfChrist: Color
+
     screen: Color
     paper: Color
-    alert: Color
+
+    black: Color
+    white: Color
+
+    darkPrimary: Color
```
Comment (rkdavidson) [inline src/data/theme/index.js:35]: Haven't looked too deeply at the existing `ThemeColors` usage, but — is the intention here that `darkPrimary` and `lightPrimary` are variations on `primary`? (e.x. navy / cyan / blue).

Or is it more  like "for the _dark_ theme, use these colors... for the _light_ theme, use these"?

Assuming the first thing ☝️, but could be misinterpreting.

Comment (asleepingpanda) [reply src/data/theme/index.js:35]: Great question! Typically, they're used as neutrals. So in the case of our current light/dark theme, `darkPrimary` is mapped to `colors.text.primary` for light mode and `lightPrimary` is mapped to `colors.text.primary` for dark mode.

**Comment by rkdavidson** [comment]
Hey @asleepingpanda is this PR still relevant, given all the Apollos 2.0 upgrading?
Should we close it?

**Comment by rkdavidson** [comment]
@asleepingpanda Bumping this PR, too — do we still need this PR given the recent theming work for web 2.0?

**Comment by asleepingpanda** [comment]
@rkdavidson I think it's probably safe to close this for the time being. I think a LOT of work will need to be done to catch it up to apollos 2.0 and it def isn't a prio for now

**Pull request closed without merging**